File: 0169-majority-element/0169-majority-element.cpp
class Solution {
public:
    int majorityElement(vector<int>& nums) {
        int left = 0 , right = nums.size() - 1 ;
        int mid = (left + right) / 2 ; 
        sort(nums.begin() , nums.end());
        return nums[mid]; 
    }
};

File: 0169-majority-element/0169-majority-element.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        count = {}
        res , Maxcount = 0 ,0
        for n in nums:
            count[n] = 1 + count.get(n,0)
            res = n if count[n] > Maxcount else res 
            Maxcount = max(count[n],Maxcount)
        return res     
        